jdaviz/app.py

```diff
@@ -2254,6 +2255,10 @@ def load_configuration(self, path=None, config=None):
         # give the vue templates access to the current config/layout
         self.config = config['settings'].get('configuration', 'unknown')
         self.vdocs = 'latest' if 'dev' in __version__ else 'v'+__version__
+        if self.config != 'unknown':
+            self.docs_link = f'https://jdaviz.readthedocs.io/en/{self.vdocs}/{self.config}/index.html'  # noqa
+        else:
+            self.docs_link = 'https://jdaviz.readthedocs.io'
```

Comment:
that is the default if nothing overrides this, yes.  https://github.com/spacetelescope/lcviz/pull/40 will override it for lcviz.  (I'm trying to avoid hardcoding anything for lcviz within jdaviz, as much as possible)

Reply:
So this PR is technically not needed for lcviz since you overwrite it downstream anyway?

Reply:
this PR moves the logic from the template into `app.py` _so that_ it can be customized downstream.

Reply:
Ah, right, I didn't see the rest of the diff, haha. In that case, you should compare against a preset list of configs, since the JS did.

Reply:
ok, I refactored to use a shared list of expected configs between this and the launcher (so the list isn't hardcoded in multiple places).